src/tools/crossgen2/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2989,7 +3008,9 @@ private ushort getRelocTypeHint(void* target)
         }
 
         private void getModuleNativeEntryPointRange(ref void* pStart, ref void* pEnd)
-        { throw new NotImplementedException("getModuleNativeEntryPointRange"); }
+        {
```

Comment:
The existing pattern (one-liner throw for things that are not needed) is replicated all over this file: I would prefer this stays consistent with the 50 other instances of this in this file.

Reply:
You are right.